client/landing/jetpack-cloud/lib/contact-support-url.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Internal dependencies
+ */
+import { JETPACK_CONTACT_SUPPORT } from 'lib/url/support';
+import { addQueryArgs } from 'lib/url';
+
+/**
+ * Creates a URL that refers to the Jetpack 'Contact Support' page,
+ * with accompanying useful information about the reason for the
+ * support request.
+ *
+ * @param {string} [siteUrl]	A site URL
+ * @param {string} [scanState]	The current state of Jetpack Scan/Backup
+ * @returns {string} 			The support request URL
+ */
+export default function contactSupportUrl( siteUrl, scanState ) {
+	// NOTE: This is an extremely simple, early implementation;
+	// if we add more options in the future, we'll want
+	// to come back and change how we build this query string
+	return addQueryArgs(
+		{
+			url: siteUrl,
+			'scan-state': scanState,
```

Comment:
Why is the code passing `scan-state`?

Reply:
Thanks for pointing this out -- I just realized it can actually be removed now. As recently as a few days ago we had some links that included `scan-state` as a query parameter and I was trying to preserve those, but now that we're getting scan state from the API they're no longer necessary. 👍

Reply:
We still have a few links that pass scan-state to the support page. Is it useful in that context? I'm not sure, was only trying to preserve existing behavior.

Reply:
Interesting. I am unsure why we're preserving that state, but can't hurt I suppose? There's nothing I wrote on the other side that looks for that, though :)